docs/credstores.md

```diff
@@ -209,6 +209,9 @@ export GCM_CREDENTIAL_CACHE_OPTIONS="--timeout 300"
 git config --global credential.cacheOptions "--timeout 300"
 ```
 
+As for _Windows_, if you enable `git.exe credential-cache` on a Windows version that supports Unix sockets, it will work.
```

Comment:
I think for now, it may be better to drop this paragraph. We can re-visit when Git for Windows official builds support versions of Windows that enable Unix sockets.

There is also some code that should be changed in `CredentialStore.cs`
```
diff --git a/src/shared/Core/CredentialStore.cs b/src/shared/Core/CredentialStore.cs
index 812c301e..bf8c0be4 100644
--- a/src/shared/Core/CredentialStore.cs
+++ b/src/shared/Core/CredentialStore.cs
@@ -152,8 +152,11 @@ namespace GitCredentialManager
                     Environment.NewLine, StoreNames.Gpg);
             }
 
-            sb.AppendFormat("  {1,-13} : Git's in-memory credential cache{0}",
-                Environment.NewLine, StoreNames.Cache);
+            if (!PlatformUtils.IsWindows())
+            {
+                sb.AppendFormat("  {1,-13} : Git's in-memory credential cache{0}",
+                    Environment.NewLine, StoreNames.Cache);
+            }
 
             sb.AppendFormat("  {1,-13} : store credentials in plain-text files (UNSECURE){0}",
                 Environment.NewLine, StoreNames.Plaintext);
@@ -286,6 +289,15 @@ namespace GitCredentialManager
 
         private void ValidateCredentialCache(out string options)
         {
+            if (PlatformUtils.IsWindows())
+            {
+                throw new Exception(
+                    $"Can not use the '{StoreNames.Cache}' credential store on Windows due to lack of UNIX socket support in Git for Windows." +
+                    Environment.NewLine +
+                    $"See {Constants.HelpUrls.GcmCredentialStores} for more information."
+                );
+            }
+
             // allow for --timeout and other options
             if (!_context.Settings.TryGetSetting(
                 Constants.EnvironmentVariables.GcmCredCacheOptions,
```

Reply:
> I think for now, it may be better to drop this paragraph. We can re-visit when Git for Windows official builds support versions of Windows that enable Unix sockets.

Do you mean this?
Of course, update will be required, but I thought it would be more helpful to include information on Windows.

````diff
export GCM_CREDENTIAL_CACHE_OPTIONS="--timeout 300"
# or
git config --global credential.cacheOptions "--timeout 300"
```
-
-As for _Windows_, if you enable `git.exe credential-cache` on a Windows version that supports Unix sockets, it will work.
-For this, you need build `git.exe` with [the instructions](https://github.com/git-for-windows/git/wiki/Building-Git) to override [`NO_UNIX_SOCKETS`](https://github.com/git-for-windows/git/blob/v2.36.1.windows.1/config.mak.uname#L637) on Windows 10 build 17061 or later.

## Plaintext files
````

---

> There is also some code that should be changed in CredentialStore.cs

Thank you. I also agree that it would be better in that it would display a message to users.
But if so, won't users be able to use the cache if they build their own `credential-cache`? Is it no problem? (I am not a maintainer of this project, so I cannot judge...)

Reply:
I fixed them. Please re-review.